,python,x4090,02.08.2024 01:32,file:///home/python/.config/libreoffice/4;